integration-tests/api-tests/specs/api-artifact-spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+//jasmine specs/api-artifact-spec.js
+
+const aetsApi = require('../api/api.js');
+const helper = require('./helpers/spec-helper.js');
+const testDataJson = require('../data/test_data/api-data.json');
+const chakram = require('chakram');
+const chakramExpect = chakram.expect;
+const jasmineExpect = expect;
+
+const HTTP_200_OK = 200;
+const getArtifactByArtifactIdData = testDataJson.getArtifactByArtifactId;
+const getMetadataBySuiteNameData = testDataJson.getMetadataBySuiteName;
+const NoExistingSuiteData = testDataJson.getArtifactByArtifactId.errorPath;
+
+describe('Get artifact by artifact Id - validation', () => {
+
+    it('Status code and JSON schema validation', () => {
+        return aetsApi.getMetadata(getMetadataBySuiteNameData.parameters).then((response) => {
```

Comment:
I think that 'then' should be mover to new line.

Reply:
ok, fixed in all files